pkg/site/site.go

```diff
@@ -56,24 +56,25 @@ const (
 	SiteConfigPrometheusServerPodAnnotationsKey string = "prometheus-server-pod-annotations"
 
 	// router options
-	SiteConfigRouterConsoleKey             string = "router-console"
-	SiteConfigRouterLoggingKey             string = "router-logging"
-	SiteConfigRouterDebugModeKey           string = "router-debug-mode"
-	SiteConfigRouterCpuKey                 string = "router-cpu"
-	SiteConfigRouterMemoryKey              string = "router-memory"
-	SiteConfigRouterCpuLimitKey            string = "router-cpu-limit"
-	SiteConfigRouterMemoryLimitKey         string = "router-memory-limit"
-	SiteConfigRouterAffinityKey            string = "router-pod-affinity"
-	SiteConfigRouterAntiAffinityKey        string = "router-pod-antiaffinity"
-	SiteConfigRouterNodeSelectorKey        string = "router-node-selector"
-	SiteConfigRouterMaxFrameSizeKey        string = "xp-router-max-frame-size"
-	SiteConfigRouterMaxSessionFramesKey    string = "xp-router-max-session-frames"
-	SiteConfigRouterDataConnectionCountKey string = "router-data-connection-count"
-	SiteConfigRouterIngressHostKey         string = "router-ingress-host"
-	SiteConfigRouterServiceAnnotationsKey  string = "router-service-annotations"
-	SiteConfigRouterPodAnnotationsKey      string = "router-pod-annotations"
-	SiteConfigRouterLoadBalancerIp         string = "router-load-balancer-ip"
-	SiteConfigRouterDisableMutualTLS       string = "router-disable-mutual-tls"
+	SiteConfigRouterConsoleKey                  string = "router-console"
+	SiteConfigRouterLoggingKey                  string = "router-logging"
+	SiteConfigRouterDebugModeKey                string = "router-debug-mode"
+	SiteConfigRouterCpuKey                      string = "router-cpu"
+	SiteConfigRouterMemoryKey                   string = "router-memory"
+	SiteConfigRouterCpuLimitKey                 string = "router-cpu-limit"
+	SiteConfigRouterMemoryLimitKey              string = "router-memory-limit"
+	SiteConfigRouterAffinityKey                 string = "router-pod-affinity"
+	SiteConfigRouterAntiAffinityKey             string = "router-pod-antiaffinity"
+	SiteConfigRouterNodeSelectorKey             string = "router-node-selector"
+	SiteConfigRouterMaxFrameSizeKey             string = "xp-router-max-frame-size"
+	SiteConfigRouterMaxSessionFramesKey         string = "xp-router-max-session-frames"
+	SiteConfigRouterDataConnectionCountKey      string = "router-data-connection-count"
+	SiteConfigRouterIngressHostKey              string = "router-ingress-host"
+	SiteConfigRouterServiceAnnotationsKey       string = "router-service-annotations"
+	SiteConfigRouterPodAnnotationsKey           string = "router-pod-annotations"
+	SiteConfigRouterLoadBalancerIp              string = "router-load-balancer-ip"
+	SiteConfigRouterDisableMutualTLS            string = "router-disable-mutual-tls"
+	SiteConfigRouterForceRestartsForHostAliases string = "router-force-restarts-for-host-aliases"
```

Comment:
Is this new site option really needed? Is there a case where the token might have aliases added to it, but you don't want the deployment to be updated?

Reply:
As well, yes yes. We can remove it, no problem at all.